src/index.js

```diff
@@ -39,6 +40,8 @@ export function getSimulator(
   return {
     request: request(app),
     render: render(app),
+    // $FlowFixMe
+    getService: token => app.getService(token),
   };
 }
 
```

Comment:
This feels like it subsumes the use case for the `getService` helper below.  Should we remove that and make this a breaking change?

Reply:
Maybe in the future we can do that, but for now lets do this with a non-breaking change